draft-ietf-quic-transport.md

```diff
@@ -1773,75 +1793,9 @@ a transport parameter therefore disables any optional protocol feature that is
 negotiated using the parameter.
 
 New transport parameters can be registered according to the rules in
-{{iana-transport-parameters}}.
-
-
-### Version Negotiation Validation {#version-validation}
```

Comment:
Build failure is because you've removed this section without updating the other places in the draft that reference it.  (Incidentally, the pre-commit hooks should have blocked you from committing something that doesn't build -- can you run `make update` to make sure that's installed?)